x-pack/plugins/triggers_actions_ui/public/application/sections/common/components/alert_quick_edit_buttons.tsx

```diff
@@ -147,72 +148,74 @@ export const AlertQuickEditButtons: React.FunctionComponent<ComponentOpts> = ({
   }
 
   return (
-    <Fragment>
+    <EuiContextMenuPanel>
       {!allAlertsMuted && (
-        <EuiButtonEmpty
+        <EuiContextMenuItem
           onClick={onmMuteAllClick}
           isLoading={isMutingAlerts}
           isDisabled={isPerformingAction}
           data-test-subj="muteAll"
         >
           <FormattedMessage
             id="xpack.triggersActionsUI.sections.alertsList.bulkActionPopover.muteAllTitle"
-            defaultMessage="Mute"
+            defaultMessage="Mute all"
```

Comment:
There was conversations with @gchaps a while ago about not using the "all" suffix here.

Reply:
If using the word "all" adds clarity, then go ahead and use it.  Otherwise, it's just redundant.